tools/check-runners

```diff
@@ -17,9 +18,12 @@ runner_obj = None
 
 runners_ok = []
 
+if 'RUNNERS_DIR' in os.environ:
+    sys.path.insert(1, os.path.abspath(os.environ['RUNNERS_DIR']))
+
 for runner in args.runners:
     try:
-        module = import_module('runners.' + runner)
```

Comment:
If this needed to be modified then what about [tools/runner](https://github.com/SymbiFlow/sv-tests/blob/master/tools/runner#L46)? I see that it still uses the old way of importing runner module.

Reply:
Absolutely right, thanks for noticing this!